resources/assets/js/app/views/votes/VoteCommentItem.js

```diff
@@ -14,5 +22,10 @@ module.exports = Marionette.ItemView.extend({
                 user: this.model.getMeta()[id].user
             }
         };
-    }
+    },
+    replyComment: function () {
+        var voteCommentAdd = new VoteCommentItemAdd();
+        $(voteCommentAdd.ui.text).val('>>> '+this.model.attributes.content_origin+'\n');
+        $(voteCommentAdd.ui.text).focus();
+    },
```

Comment:
Please create new method in VoteCommentItemAdd() for content setting and for focus and just call them.

Why are you initialize new view?